.github/workflows/orca.yml

```diff
@@ -11,7 +11,7 @@ on:
         -   cron: "0 0 * * 0"
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
```

Comment:
What's the reason for this?

Reply:
This is required for D11 jobs as it requires an updated version of SQLite which is only available with ubuntu-24.04.

Reply:
TIL "latest" doesn't actually mean "latest" (`latest` is currently 22.04): https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories

Reply:
Ah, the old `$x != $x` problem. 😅 Noted. Thanks, @danepowell. 😛